src/components/CallView/CallView.vue

```diff
@@ -417,7 +415,6 @@ export default {
 		// Ensure that data is properly initialized before mounting the
 		// subviews.
 		this.updateDataFromCallParticipantModels(this.callParticipantModels)
-		this.isBackgroundBlurred = BrowserStorage.getItem('background-blurred') !== 'false'
```

Comment:
should we still read this? Otherwise there will be a performance regression on "update"

Reply:
I would say, we can read it once, and if value exists in the storage, force update theming on server via new API, then drop value from storage?

Reply:
sounds good to me